src/geo/map/cartodb-layer.js

```diff
@@ -31,6 +45,17 @@ var CartoDBLayer = LayerModelBase.extend({
     return this.get('visible');
   },
 
+  getGeometryType: function () {
```

Comment:
this method basically works because of the cartodb wizards work but you can't guess geometry type from style